src/redis-cli.c

```diff
@@ -1088,6 +1088,7 @@ static int parseOptions(int argc, char **argv) {
         } else if (!strcmp(argv[i],"-n") && !lastarg) {
             config.dbnum = atoi(argv[++i]);
         } else if (!strcmp(argv[i],"-a") && !lastarg) {
+            fputs("Warning: Using a password on the command line interface can be insecure.\n", stderr);
```

Comment:
I hope this isn't trademarked by MySQL :)

Reply:
Yes, it's originated from MySQL(#^.^#). I didn't notice that the warning message may also involved in trademark issues. So i changed the message a little bit and i think now it's fine.